label_studio/jwt_auth/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from . import views
+
+app_name = 'jwt_auth'
+
+urlpatterns = [
+    path('api/jwt/settings', views.JWTSettingsAPI.as_view(), name='api-jwt-settings'),
+    path('api/token/', views.LSAPITokenView.as_view(), name='token_manage'),
```

Comment:
try to just override this in LSE, to see if we can avoid the awkward configuration for serializers

Reply:
/projects/list apparently does a similar thing